spotify-artist-viewer/Models/SAArtistTableViewController.m

```diff
@@ -42,12 +41,16 @@ - (void)didReceiveMemoryWarning {
 #pragma mark - Text Change
 
 -(void)searchBar:(UISearchBar *)searchBar textDidChange:(NSString *)searchText{
+  
     SARequestManager *manager = [SARequestManager sharedManager];
     
-
     [manager getArtistWithQuery:searchText success:^(NSArray *artists)  {
-       [self.tableView reloadData];
-        self.artistArray = artists;
+        NSLog(@"success");
```

Comment:
This is ok for test code but keep in mind that NSLogs should not be include in code that will be merged into production code.